fbrp/src/fbrp/runtime/conda.py

```diff
@@ -108,7 +108,9 @@ async def run_cmd_with_envvar(self, conda_env):
             executable="/bin/bash",
             cwd=self.proc_def.root,
             env=conda_env,
+            preexec_fn=os.setsid,
         )
+        self.proc_pgrp = os.getpgid(self.proc.pid)
```

Comment:
handle the case process died before this line executed.

Reply:
TODO for now, unless you know an easy way to do this

Reply:
Shamelessly copying from stackoverflow, would this help ? https://stackoverflow.com/questions/568271/how-to-check-if-there-exists-a-process-with-a-given-pid-in-python

Reply:
ack